build_projects/dotnet-cli-build/build.sh

```diff
@@ -41,12 +41,11 @@ while [[ $# > 0 ]]; do
             echo ""
             echo "Options:"
             echo "  --configuration <CONFIGURATION>     Build the specified Configuration (Debug or Release, default: Debug)"
-            echo "  --targets <TARGETS...>              Comma separated build targets to run (Init, Compile, Publish, etc.; Default is a full build and publish)"
-            echo "  --nopackage                         Skip packaging targets"
             echo "  --skip-prereqs                      Skip checks for pre-reqs in dotnet_install"
+            echo "  --nopackage                         Skip packaging targets"
```

Comment:
To make it consistent with [the Usage line above](https://github.com/dotnet/cli/pull/3435/files/33f61e7dcf02d14e81ea7a2a405ff076d1fe8669#diff-77fc6850a27234595d773b7c80f2280eR40).